### DISCUSSION TITLE
How to press two keys at once? eg. Ctrl + A/S

### DISCUSSION THREAD

**Discussion by alexwideman** [seleniumbase/SeleniumBase#2699] [Q&A] [answered]
I'm using uc mode which is working well, thank you Michael!

Once I get through to the PDF document, I've tried saving it with the following two approaches, but the PDFs are corrupted/won't open.

```
pdf_url = sb.get_attribute("a#FullScreen", "href")
sb.save_file_as(pdf_url, document_title + ".pdf", destination_folder=DOWNLOAD_DIR)

sb.driver.uc_click("a#FullScreen", 3)
sb.download_file(pdf_url, destination_folder=DOWNLOAD_DIR)
```

To avoid this, I want to mimic user behaviour by pressing CTRL+S. Is there a way to do it with press_keys? I have looked on the website and Github discussions and haven't be able to figure it out. Thanks!

**Comment by mdmintz** [comment, marked as answer]
The CTRL+S is an action that takes place outside the web page itself, which means you need to use something like `pyautogui` to do that, as mentioned on Stack Overflow here: https://stackoverflow.com/a/63055535/7058266

**Comment by pol4xer** [comment]
I had to use "paste" and this worked for me:
```python3
element = driver.find_element(...)
# Mac
element.send_keys(Keys.COMMAND + "v")
# Windows
element.send_keys(Keys.CONTROL + "v")
```

Maybe it can help you with your problem
